### ISSUE TITLE
Feature request: Extract chimeric read pairs

### ISSUE THREAD

**Issue by etvedte** [broadinstitute/picard#1344]
### Tool(s) involved
CollectAlignmentSummaryMetrics

### Description
CollectAlignmentSummaryMetrics currently reports the percentage of reads meeting various criteria defining it as a putative chimeric read. Would it be possible to add functionality to extract these reads? Performing a grep on the BAM file for the 'SA' tag would only retrieve a subset of chimeras reported by this tool.

**Comment by yfarjoun** [comment]
So you would like the tool to optionally write out chimeric reads? Hmmm. Seems out of place in this tool which collects all sorts of alignment metrics....I think it would be better to include it in a tool like FilterSamReads....what do you think?

**Comment by etvedte** [comment]
True. I'm interested in the feature, regardless of the tool it's associated with. I appreciate the prompt response!

**Comment by yfarjoun** [comment]
do you think you'd be able to submit a Pull Request modifying FilterSamFile using the ChimeraUtil.isChimeric() method?
